e2e/wdio/core/tests/object-identifier.e2e-spec.ts

```diff
@@ -0,0 +1,54 @@
+import { ObjectIdentifierPo } from '../pages/object-identifier.po';
+import {
+    click,
+    getElementAriaLabel,
+    getElementArrayLength,
+    getElementTitle,
+    refreshPage,
+    scrollIntoView,
+    waitForPresent
+} from '../../driver/wdio';
+import { checkElArrIsClickable } from '../../helper/assertion-helper';
+
+
+describe('Object identifier test suite', function() {
+    const objectIdentifierPage = new ObjectIdentifierPo();
+    const {
+        identifier,
+        clickableLinks,
+    } = objectIdentifierPage;
+
+    beforeAll(() => {
+        objectIdentifierPage.open();
+    }, 1);
+
+    it('Verify each marker is clickable', () => {
+        checkElArrIsClickable(identifier);
+    });
+
+    afterEach(() => {
+        refreshPage();
+        waitForPresent(identifier);
+    }, 1);
+
+    it('Verify each identifier is clickable', () => {
+        const arr = getElementArrayLength(identifier);
+        for (let i = 0; i < arr; i++) {
+            scrollIntoView(identifier, i);
+            click(identifier, i);
```

Comment:
seems like you need to check 'clickableLinks' not 'identifier' 
also,  you'll want to use isElementClickable('clickableLinks, i); instead of click() becuase click will activate the link navigation and can't be used like this in a loop. you'd need a separate test case to check navigation